pkg/adapter/v2/cloudevents.go

```diff
@@ -218,8 +219,6 @@ func (c *client) reportError(reportArgs *source.ReportArgs, result protocol.Resu
 			result = fmt.Errorf("%w\nmetrics reporter error: %s", result, rErr)
```

Comment:
this does nothing since you don't return result anymore, which is good change IMO. Instead what about logging the error?

Reply:
absolutely right.. I will make those changes.. thanks for reviewing :)